packages/timeline-state-resolver/src/integrations/atem/index.ts

```diff
@@ -179,23 +176,9 @@ export class AtemDevice
 		// Make sure there is something to diff against
 		oldAtemState = oldAtemState ?? this._atem.state ?? AtemStateUtil.Create()
 
-		// bump out any auxes that we don't control as they may be used for CC etc.
-		const noOfAuxes = Math.max(oldAtemState.video.auxilliaries.length, newAtemState.video.auxilliaries.length)
-		const auxMappings = Object.values<Mapping<unknown>>(mappings)
-			.filter(
-				(mapping: Mapping<SomeMappingAtem>): mapping is Mapping<MappingAtemAuxilliary> =>
-					mapping.options.mappingType === MappingAtemType.Auxilliary
-			)
-			.map((mapping) => mapping.options.index)
-
-		for (let i = 0; i < noOfAuxes; i++) {
-			if (!auxMappings.includes(i)) {
-				oldAtemState.video.auxilliaries[i] = undefined
-				newAtemState.video.auxilliaries[i] = undefined
-			}
-		}
+		const diffOptions = createDiffOptions(mappings)
```

Comment:
Question: Should we add a device option for enabling/disabling partial control?

Reply:
This is currently intended to behave identically to previous versions of TSR in what it will control.
Long term, we should make this more configurable, but I don't want to do that as part of this PR.